Lib/_collections_abc.py

```diff
@@ -409,6 +413,80 @@ def __subclasshook__(cls, C):
         return NotImplemented
 
 
+class _CallableGenericAlias(GenericAlias):
+    """ Internal class specifically for consistency between the ``__args__`` of
+    ``collections.abc.Callable``'s and ``typing.Callable``'s ``GenericAlias``.
+    """
+    __slots__ = ()
+    def __new__(cls, origin, args):
+        try:
+            return cls.__getitem_type(origin, args)
+        except TypeError:
```

Comment:
Is this *just* intended to catch the two explicit `raise` statements in `__getitem_type()` below? Then why spend any time in that function producing a nice error message? Why not just write `return GenericAlias(origin, args)` there?